app/views/users/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1> Welcome to Users List</h1>
\ No newline at end of file
+<div class = 'user'>
+<h1> Welcome to Users List</h1>
+<div class = 'user_container'>
+<% @users.each do |user|%>
+<div class = 'user_row'>
+<div class = 'user_image'>
+<img src = "https://unsplash.com/s/photos/computer">
+</div>
+<div class = 'user_block'>
+<h3><%= link_to user.name, user_path(user)  %></h3>
+<img src="<%= link_to user.photo, %>" >
+<p>Number of Posts = <%=user.posts.count %></p>
```

Comment:
- [ ] There is a syntax error on line 11, you have a comma after `user.photo`. When you fix it correct another error on line 12. Kindly make use of the model methods you created in the previous milestones to display the count of user posts. I strongly advise you to test the app and make sure it works before submitting the PR for another review. __I did not test the app for adding posts, comments, and likes because of the errors__.